code/modules/antagonists/changeling/powers/tiny_prick.dm

```diff
@@ -255,15 +255,26 @@
 	name = "Hallucination Sting"
 	desc = "We cause mass terror to our victim. Costs 10 chemicals."
 	helptext = "We evolve the ability to sting a target with a powerful hallucinogenic chemical. \
-			The target does not notice they have been stung, and the effect occurs after 30 to 60 seconds."
+			The target does not notice they have been stung, and they view everyone else as a random passerby."
 	button_icon_state = "sting_lsd"
 	chemical_cost = 10
 	dna_cost = 1
 
 /datum/action/changeling/sting/lsd/sting_action(mob/user, mob/living/carbon/target)
 	..()
 	log_combat(user, target, "stung", "LSD sting")
-	addtimer(CALLBACK(src, PROC_REF(hallucination_time), target), rand(30 SECONDS, 60 SECONDS))
+	var/mob/random_guy = pick(oviewers(8, user))
```

Comment:
`oviewers` is people who see the user, not users the viewer sees. 

This is kind of an important distinction because I'm pretty sure this will pick ghosts.

Reply:
It should prooobably also have code to prioritize picking humans over simplemobs.